### ISSUE TITLE
Support for site owner avatar img?

### ISSUE THREAD

**Issue by asdfhamiltonian** [renyuanz/leonids#1]
Not sure if I'm missing something, but it looks like this jekyll theme doesn't include code to support an site owner avatar image. Also the example yml file references an avatar image that doesn't exist in the /img/ folder. Otherwise really cool theme design so far!

**Comment by renyuanz** [comment]
Hi @asdfhamiltonian, thank you for this first issue.
That is a good suggestion and I was thinking about it when I started the project.

As you can see, this theme is inspired by purecss blog layout and you can find the original version [here](http://purecss.io/layouts/blog/).

The original one shows author's avatar for every post and it looks pretty good if the blog has several authors, but it will be weird if the blog is single author.  

So I do like to add author's avatar at the sidebar, just above the author's name and hide it when screen gets smaller.

How do you think? I'll work on it this weekend and let you know if finished.

Have a nice day,
Renyuan

**Comment by asdfhamiltonian** [comment]
Hey Renyuan, that sounds good! It could definitely be weird to have an author photo with every post for a multi-author blog. My solution was just to paste a photo into the sidebar section and it seems to work ok for now.

Let me know how it goes, I look forward to seeing what you come up with. Thank you for coming up with this cool Jekyll layout.

Sincerely,

Max

**Comment by renyuanz** [comment]
Hi @asdfhamiltonian, 
I did make this change yesterday and it looks like this:
<img width="1237" alt="screen shot 2015-09-11 at 09 32 34" src="https://cloud.githubusercontent.com/assets/10615036/9809401/2ba99c28-5868-11e5-8f27-a06cae4b03ba.png">

I couldn't come up with a better design at the moment, but if you have any idea just share it here, I'll really appreciate it. :-)

I'll just push it so everyone could see the live demo.

Cheers,
Renyuan

**Comment by asdfhamiltonian** [comment]
Hi @renyuan,

The update looks cool! I’ll try updating my page to the new design and let you know if I think of anything.

Regards,

Max

**Comment by asdfhamiltonian** [comment]
Hey Renyuan,

I thought I should update you on some feedback I got about the Jekyll 
theme. My career advisor for this code school I use said that their 
designer didn't like the idea of having the author's photo on the 
sidebar (turns out that maybe my idea for this was bad). What I did 
instead is add a photo to my about page. They also thought that the 
icons for github, linkedin etc. were a bit too small. Hope this 
information helps and let me know what you think!

Best,

Max
